tests/tests/test_mender_connect.py

```diff
@@ -17,14 +17,12 @@
 import time
 
 from testutils.api import proto_shell
-from testutils.api.proto_shell import WRONG_BEHAVIOR
 from ..common_setup import class_persistent_standard_setup_one_client_bootstrapped
 from ..MenderAPI import devconnect
 
-if WRONG_BEHAVIOR:
-    logger = logging.getLogger("websockets")
-    logger.setLevel(logging.DEBUG)
-    logger.addHandler(logging.StreamHandler())
+logger = logging.getLogger("websockets")
+logger.setLevel(logging.DEBUG)
+logger.addHandler(logging.StreamHandler())
```

Comment:
Oh, and this too, btw. It is *extremely* verbose, so I think it is too much for regular logs after this development has finished.

Reply:
ack